importer/tasks.py

```diff
@@ -239,14 +240,33 @@ def get_item_info_from_result(result):
     return m.group(1), item_url
 
 
-async def import_item_count_from_url(import_url):
+def fetch_all_urls(items):
+    with concurrent.futures.ThreadPoolExecutor(max_workers=15) as executor:
+        result = executor.map(import_item_count_from_url, items)
+    finals = []
+    totals = 0
+
+    for value, score in result:
+        totals = totals + score
+        finals.append(value)
+
+    return finals, totals
+
+
+def import_item_count_from_url(import_url):
     """
     Given a loc.gov URL, return count of files from the resources section
     """
-    resp = requests.get(import_url, params={"fo": "json"})
-    resp.raise_for_status()
-    item_data = resp.json()
-    return len(item_data["resources"][0]["files"])
+    try:
+
+        resp = requests.get(import_url, params={"fo": "json"})
+        resp.raise_for_status()
+        item_data = resp.json()
+        output = len(item_data["resources"][0]["files"])
+        return f"{import_url} {output}", output
```

Comment:
The return signature for this function seems a bit inconsistent with it either being a string containing the URL and the count (which would need to be parsed), followed by the count or the string value of an exception and the count.

It might make sense to add an explicit success value and clarify that the intention is to return a human-meaningful message. 

You can add a type annotation to have tools like mypy or pylance (in VSCode) validate these calls:

```python
from typing import Tuple

def import_item_count_from_url(import_url: str) -> Tuple[bool, str, int]:
```